opensrp-reveal/src/main/res/values/dimens.xml

```diff
@@ -17,6 +17,7 @@
     <dimen name="drawer_separator_margin">20dp</dimen>
     <dimen name="drawer_drawable_padding">10dp</dimen>
     <dimen name="dialog_min_width_major">90%</dimen>
+    <dimen name="operator_top_margin">50dp</dimen>
```

Comment:
Don't you need a dimension for tablets as well?

Reply:
This is minimum margin and the value is used for all devices. Constraint  vertical weight and bias are used to position the view in bigger screens or when normal or small text is used in the mobile devices(space is available).